pkg/component/extensions/infrastructure/infrastructure.go

```diff
@@ -254,9 +257,15 @@ func (i *infrastructure) NodesCIDR() *string {
 	return i.nodesCIDR
 }
 
+// EgressCIDRs returns a list of CIDRs used as source IP by any traffic originating from the shoot's worker nodes.
+func (i *infrastructure) EgressCIDRs() []string {
+	return i.egressCIDRs
+}
+
 func (i *infrastructure) extractStatus(status extensionsv1alpha1.InfrastructureStatus) {
 	i.providerStatus = status.ProviderStatus
 	i.nodesCIDR = status.NodesCIDR
+	i.egressCIDRs = status.DeepCopy().EgressCIDRs
```

Comment:
```suggestion
	copy(i.egressCIDRs, status.EgressCIDRs)
```
nit: you don't need to DeepCopy the whole status, but the slice only.

Reply:
PTAL :)